docs/quickstart.md

```diff
@@ -163,12 +178,7 @@ We can deploy the subnet nodes. Note that each node should be importing a differ
 
 ## Step 7: Configure the IPC agent
 
-For ease of use, we'll import the remaining keys into the first validator, via which the IPC Agent will act on behalf of all.
-
-* Copy the wallet keys into the docker container and import them
-```bash
-docker cp ~/.ipc-agent/wallet2.key <CONTAINER_NAME_1>:/input.key && docker exec -it <CONTAINER_NAME_1> eudico wallet import --format=json-lotus input.key
-docker cp ~/.ipc-agent/wallet3.key <CONTAINER_NAME_1>:/input.key && docker exec -it <CONTAINER_NAME_1> eudico wallet import --format=json-lotus input.key
+For ease of use, we'll import the remaining keys into the first validator, via which the IPC Agent will act on behalf of all. All the keys should already be imported in the wallet of the agent, if for some reason this is not the case you can import it through: `./ipc-agent/bin/ipc-agent wallet import --path=~/.ipc-agent/wallet.key`
 ```
```

Comment:
```suggestion
For ease of use, we'll import the remaining keys into the first validator, via which the IPC Agent will act on behalf of all.
```

There was a stray code block opening here. I also don't think we need the import instructions here as you should never run into that scenario when following the instructions.